dolphin/api/v1/storages.py

```diff
@@ -53,6 +53,15 @@ def validate_parameters(data, required_parameters,
             raise exc_response(explanation=msg)
 
 
+def update_access_info(access_info, body):
+    for attr, value in access_info.__dict__.items():
+        if attr in body:
+            if attr == 'extra_attributes':
+                access_info.extra_attributes.update(body['extra_attributes'])
+            access_info[attr] = body[attr]
+    del access_info.created_at, access_info.updated_at
```

Comment:
Why delete updated_at here ?

Reply:
created_at and updated_at will inserted automatically by sqlachemy session